cmd/agent-test/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"sigs.k8s.io/yaml"
+
+	"github.com/containers/nri-plugins/pkg/agent"
+	cfgapi "github.com/containers/nri-plugins/pkg/apis/config/v1alpha1"
+	logger "github.com/containers/nri-plugins/pkg/log"
+)
+
+var (
+	log        = logger.Get("agent-test")
```

Comment:
I'm wondering if it would be better to keep all the tests under root `test/` directory?

Reply:
Sorry, I pushed this accidentally. It was not supposed to be part of the PR at all.